src/core_plugins/tile_map/public/editors/tile_map_vis_params.js

```diff
@@ -0,0 +1,13 @@
+import { uiModules } from 'ui/modules';
+import tileMapTemplate from './tile_map_vis_params.html';
+import './wms_options';
+const module = uiModules.get('kibana');
+module.directive('tileMapVisParams', function () {
+  return {
+    restrict: 'E',
+    template: tileMapTemplate,
+    link: function () {
```

Comment:
* Get rid of the `link` function if we don't need it.
* Use child scope for the directive instead of writing to `vis.params` directly in the template.

Reply:
+1